File: Week01/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = height - 1; i >= 0; i--)
    {
        for (int j = 0; j < height; j++)
        {
            if (j >= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}

File: Week06/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    height = get_height()
    for i in range(1, height + 1):
        print(" " * (height - i) + "#" * i, end="  ")
        print("#" * i)


def get_height():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            return height


main()
